src/sentry/lang/javascript/processor.py

```diff
@@ -187,18 +187,32 @@ def fetch_release_file(filename, release):
     cache_key = 'releasefile:v1:%s:%s' % (
         release.id,
         md5(filename).hexdigest(),
-    )
+    ),
+
+    filename_path = None
+    try:
+        # Can we guarantee filename is always parsable?
+        parsed_url = urlparse(filename)
+        filename_path = parsed_url.path
+    except Exception:
+        pass
+
     logger.debug('Checking cache for release artifact %r (release_id=%s)',
                  filename, release.id)
     result = cache.get(cache_key)
+
     if result is None:
         logger.debug('Checking database for release artifact %r (release_id=%s)',
                      filename, release.id)
-        ident = ReleaseFile.get_ident(filename)
+
+        filename_idents = [ReleaseFile.get_ident(filename)]
+        if filename_path is not None:
+            filename_idents.append(ReleaseFile.get_ident(filename_path))
+
         try:
             releasefile = ReleaseFile.objects.filter(
                 release=release,
-                ident=ident,
+                ident__in=filename_idents,
             ).select_related('file').get()
```

Comment:
This `.get()` now is going to be incorrect since it's possible to get two responses back. So we need to handle this case then implement logic to pick the right one.